src/Catty/DataModel/Configuration/PenConfiguration.swift

```diff
@@ -24,17 +24,37 @@ struct PenConfiguration {
     var penDown = false
     static let sizeConversionFactor = 0.634
```

Comment:
One more thing: can we use `CGFloat` here directly instead having to convert data types at every call?

Reply:
Yeah, that's true. Thanks!
I have pushed with that change. Please review it.